src/polyswarm/client/report_template.py

```diff
@@ -0,0 +1,161 @@
+@click.argument('template-id', callback=utils.validate_id)
+@click.option('--template-name', type=click.STRING)
+@click.option('--primary-color', type=click.STRING)
+@click.option('--footer-text', type=click.STRING)
+@click.option('--last-page-text', type=click.STRING)
+@click.option('--last-page-text-file', type=click.Path(exists=True), help='File with last page text.')
+@click.option('--includes',
+              help=f'Comma-separated list of sections to include in the report. Can be one ore more of: {SECTIONS}',
+              multiple=True,
+              callback=lambda _,o,x: x[0].split(',') if len(x) == 1 else x)
+@click.option('--excludes',
+              help=f'Comma-separated list of sections to exclude in the report. Can be one ore more of: {SECTIONS}',
+              multiple=True,
+              callback=lambda _,o,x: x[0].split(',') if len(x) == 1 else x)
+@click.pass_context
+def update(ctx, template_id, template_name, primary_color,
+           footer_text, last_page_text, last_page_text_file, includes, excludes):
+    api = ctx.obj['api']
+    output = ctx.obj['output']
+    if last_page_text_file:
+        if last_page_text:
+            raise click.BadOptionUsage('--last-page-text-file',
+                                       'Cannot use both --last-page-text and --last-page-text-file')
+        with open(last_page_text_file, 'r') as f:
+            last_page_text = f.read()
+    output.report_template(api.report_template_update(template_id=template_id,
+                                                      template_name=template_name,
+                                                      primary_color=primary_color,
+                                                      footer_text=footer_text,
+                                                      last_page_text=last_page_text,
+                                                      includes=includes if includes else None,
+                                                      excludes=excludes if excludes else None))
+
+
+@report_template.command('get', short_help='Fetch a report template.')
+@click.argument('template-id', callback=utils.validate_id)
+@click.pass_context
+def get(ctx, template_id):
+    api = ctx.obj['api']
+    output = ctx.obj['output']
+    output.report_template(api.report_template_get(template_id=template_id))
+
+
+@report_template.command('delete', short_help='Delete a report template.')
+@click.argument('template-id', type=click.INT, required=True)
+@click.pass_context
+def delete(ctx, template_id):
+    api = ctx.obj['api']
+    api.report_template_delete(template_id=template_id)
+    click.echo('Template deleted')
+
+
+@report_template.command('default', short_help='Set as default a report template.')
+@click.argument('template-id', type=click.INT, required=True)
+@click.pass_context
+def set_default(ctx, template_id):
+    api = ctx.obj['api']
+    api.report_template_default(template_id=template_id)
```

Comment:
I would replace this with `report_template_update`. Likely I would make setting the default as part of the `update` command rather than having an extra command for this.

Reply:
Removed, the default can be set in the update command.